src/mds/MDLog.cc

```diff
@@ -168,6 +168,22 @@ uint64_t MDLog::get_safe_pos() const
   return journaler->get_write_safe_pos(); 
 }
 
+estimated_reply_time_t MDLog::get_replay_estimated_complete_time(){
+  auto current_pos = journaler->get_read_pos();
+  auto total_bytes = end_pos - start_pos;
+  estimated_reply_time_t estimated_time;
+  double percent_complete = (double)  (current_pos - start_pos) / total_bytes;
+  auto time = percent_complete * \
+    std::chrono::duration_cast<std::chrono::seconds>(ceph::coarse_mono_clock::now() - start_time);
+  estimated_time.percent_complete = percent_complete * 100;
+  mds->clog->warn() <<  "_replay status " << percent_complete * 100 << "\% complete" 
```

Comment:
I think the  mons with issue a cluster log entry for each update the to the health metric. Do we need this or is it redundant?

Reply:
Isn't that only done when clearing a health warning?

Reply:
I don't think so?

Reply:
I'm looking at mon/MDSMonitor.cc and that clog's when health message is cleared. I'm missing something then...

Reply:
See `PaxosServer::encode_health` and `Monitor::log_health`.

Reply:
@batrick @vshankar Should I remove this warning?

Reply:
@batrick The log looks like this:
```
out/mon.a.log:2023-10-31T13:53:02.740+0530 7ff84d77f700 10 mon.a@0(leader).log v106  logging 2023-10-31T13:53:02.742652+0530 mds.a (mds.0) 1 : cluster [WRN] _replay status 7.57157% complete Elapsed time: 34s Estimated time remaining: 415.048 seconds
out/mon.a.log:2023-10-31T13:53:02.920+0530 7ff84af7a700  7 mon.a@0(leader).log v107 update_from_paxos applying incremental log 107 2023-10-31T13:53:02.742652+0530 mds.a (mds.0) 1 : cluster [WRN] _replay status 7.57157% complete Elapsed time: 34s Estimated time remaining: 415.048 seconds
```
Which is incremental, It does not repeat for every check.

Reply:
I have removed the log.